lib/y-richtext.coffee

```diff
@@ -215,10 +235,16 @@ class YRichText extends BaseClass
       throw new Error "This part of code must not be reached!"
 
   insertHelper : (position, content) ->
-    if content?
-      @_get("characters").insertContents position, content.split("") # convert content to an array
+    as_array =
+      if typeof content == "string"
```

Comment:
Well if I do in a REPL `1 == '1'` in **coffee**, I get false.

    $ coffee
    coffee> 1=="1"
    false